portfolio/src/main/webapp/index.html

```diff
@@ -11,56 +11,70 @@
-          <button class="mdc-button mdc-button--outlined" onclick="intro()" style="color:white;">  
-	     <span class="mdc-button__ripple">Intro</span> 
-	  </button>
+    <div id="intro-content">
+      <div id="intro-button-one">
+        <div class="mdc-touch-target-wrapper">
+          <button class="mdc-button mdc-button--outlined" onclick="intro()">  
+	        <span class="mdc-button__ripple">Intro</span> 
+	       </button>
         </div>
-        <div class="mdc-touch-target-wrapper" style="display:block;">
-           <button class="mdc-button mdc-button--outlined" onclick="tech()" style="color:white;">  
+        <div class="mdc-touch-target-wrapper">
+           <button class="mdc-button mdc-button--outlined" onclick="tech()">  
 	      <span class="mdc-button__ripple">Tech Skills</span> 
            </button>
         </div>
-        <div class="mdc-touch-target-wrapper" style="display:block;">
-	   <button class="mdc-button mdc-button--outlined" onclick="interests()" style="color:white;">  
+        <div class="mdc-touch-target-wrapper">
+	   <button class="mdc-button mdc-button--outlined" onclick="interests()">  
 	       <span class="mdc-button__ripple">Interests</span>
 	   </button>
         </div>
       </div>
-      <div id="basic-profile" style="display:inline-block;">
+      <div id="basic-profile">
         <img src="/images/PROFILE_PIC.JPG">
       	<h1>Michaela Murray's Portfolio</h1>
       	<div id="intro">
-      	  <ul style="list-style-type:none;">
+      	  <ul>
             <li>Stanford Class of 2022</li> 
             <li>Major: Computer Science</li>
             <li>STEP Intern @ Google</li>
 	  </ul>
         </div>
       </div>
-      <div id="intro-button-two" style="display:inline-block;">
-	<div class="mdc-touch-target-wrapper" style="display:block;">
-	   <button class="mdc-button mdc-button--outlined" style="color:white;">  
+      <div id="intro-button-two">
+	<div class="mdc-touch-target-wrapper">
+	   <button class="mdc-button mdc-button--outlined">  
 	      <span class="mdc-button__ripple">Blog</span>
 	   </button>
         </div>
-        <div class="mdc-touch-target-wrapper" style="display:block;">
-          <form action="/pdf/Michaela_Murray_Resume.pdf" target="_blank" style="display:inline-block;">
-            <button class="mdc-button mdc-button--outlined" style="color:white;">  
+        <div class="mdc-touch-target-wrapper">
+          <form action="/pdf/Michaela_Murray_Resume.pdf" target="_blank">
+            <button class="mdc-button mdc-button--outlined">  
 	       <span class="mdc-button__ripple">Resume</span> 
 	    </button>
           </form>
-        </div> 
-        <div class="mdc-touch-target-wrapper" style="display:block;">
-      	  <button class="mdc-button mdc-button--outlined" onclick="contact()" style="color:white;">  
-	     <span class="mdc-button__ripple">Contact</span> 
-	  </button>
+        </div>
+	<div id="icons">
+            <ul > 
+            <li > 
```

Comment:
Some of this indentation looks off again (the li is at the same indentation level as the ul). But let's not block on that for now...